content/conditions/type-2-diabetes/getting-diagnosed/main-content.md

```diff
@@ -0,0 +1,59 @@
+## Getting diagnosed
+
+Type 2 diabetes is often diagnosed following blood or urine tests for
+something else.
+
+However, if you have any [symptoms of diabetes](check-if-you-have-it) you
+should see your GP straight away.
+
+To find out if you have type 2 diabetes, you usually have to go through the
+following steps:
+
+1. Speak to your GP about your symptoms.
+2. Your GP will check your urine and arrange a blood test to check your blood
+   sugar levels. It usually takes about 1 to 2 days for the results to come back.
+3. If you have diabetes, your GP will ask you to come in again to explain the
+   test results and what will happen next.
+
+### If you’re diagnosed with diabetes
+
+What your GP will discuss with you during your appointment depends on the
+diagnosis and the treatment they recommend.
+
+Generally, they’ll talk to you about:
+
+* what diabetes is
+* what high blood sugar means for your health
+* what medication you’ll have to take
+* your diet and exercise
+* your lifestyle, for example alcohol and smoking
+
+!!! warning
+Your GP will do their best to discuss the diagnosis with you but this first
+appointment might only be 10 to 15 minutes.
+!!!
+
+### If you have questions about your diagnosis
+
+It’s usually difficult to take everything in the GP tells you during the
```

Comment:
"to take everything in the GP tells you" - should this be "take in everything the GP tells you..." ?